client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetails.tsx

```diff
@@ -45,6 +46,12 @@ const ClinicalDetails: React.FC = (): JSX.Element => {
         setHasBackgroundDiseases, setSymptoms, setBackgroundDiseases, context, setIsolationCityName, setIsolationStreetName, setStreetsInCity
     });
 
+    React.useEffect(() => {
+        tabsObserver.subscribe(id, () => {
```

Comment:
You need to cleanup the event subscription when the component leaves the DOM. Maybe call it `unsubscribe`.